android/gradle.properties

```
@@ -12,6 +12,11 @@
 # org.gradle.parallel=true
 StripeTerminalReactNative_compileSdkVersion=33
 StripeTerminalReactNative_kotlinVersion=1.8.21
-StripeTerminalReactNative_targetSdkVersion=31
+StripeTerminalReactNative_targetSdkVersion=33
 android.useAndroidX=true
 android.enableJetifier=true
+
+org.gradle.parallel=true
+org.gradle.daemon=true
+org.gradle.jvmargs=-Xmx12800M
```

Let's bring this down to something more manageable like 4GB:

```suggestion
org.gradle.jvmargs=-Xmx4096M
```